components/XGFEDatePicker.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react'
+import DatePicker from 'react-native-datepicker'
+
+export default class XGFEDatePicker extends Component {
+  render(){
+  	const {
+      input: { onBlur, onChange, onFocus, value },
+      placeholder,
+      meta: { error, toched, valid },
+      ...rest
+    } = this.props
+
+    return (
+      <DatePicker
+        style={{width: '100%'}}
+        date={value}
+        mode="date"
+        placeholder={placeholder}
+        format="YYYY-MM-DD"
+        confirmBtnText="Confirm"
+        cancelBtnText="Cancel"
+        customStyles={{
```

Comment:
Is it possible to not use absolute position for our component.

Reply:
it is possible